src/OrbitGl/TrackManager.h

```diff
@@ -29,6 +30,8 @@
 #include "Viewport.h"
 #include "capture_data.pb.h"
 
+using SystemMemoryTrack = orbit_gl::MemoryTrack<orbit_gl::kSystemMemoryTrackSize>;
+
```

Comment:
I believe this declaration should be in `MemoryTrack.h`.

Reply:
Moved to `MemoryTrack.h` now. Thanks!